packages/app-store/jelly/lib/VideoApiAdapter.ts

```diff
@@ -0,0 +1,60 @@
+import type { CalendarEvent, EventBusyDate } from "@calcom/types/Calendar";
+import type { CredentialPayload } from "@calcom/types/Credential";
+import type { PartialReference } from "@calcom/types/EventManager";
+import type { VideoApiAdapter, VideoCallData } from "@calcom/types/VideoApiAdapter";
+
+type JellyToken = {
+  access_token: string;
+  token_type: string;
+  expires_in: number;
+  refresh_token: string;
+  scope: string;
+};
+const JellyVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter => {
+  return {
+    createMeeting: async (event: CalendarEvent): Promise<VideoCallData> => {
+      // get keys from slug
+      const keys = credential.key as JellyToken;
+      console.log("jelly API keys: ", keys);
+      const { access_token } = keys;
+      console.log("jelly access token: ", access_token);
+      // create jelly link
+      const jellyLink = await fetch("https://jellyjelly.com/api/ti/start_jelly", {
+        method: "POST",
+        body: JSON.stringify({ event }),
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${access_token}`,
+        },
+      });
```

Comment:
I can't create new links, I'm getting this error `"name":"RequestContentLengthMismatchError","code":"UND_ERR_REQ_CONTENT_LENGTH_MISMATCH","message":"Request body length does not match content-length header"`

Reply:
Pretty sure this is fixed now, feel free to give things a try again!